### PR TITLE
egrep command to verify method of boot changed

### PR DESCRIPTION
**Pull request by SavioJoshua**
## Summary and Scope

_Summarize what has changed. Explain why this PR is necessary. What is impacted? Is this a new feature, critical bug fix, etc?_

_Is this change backwards incompatible, backwards compatible, or a backwards compatible bugfix?_

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [issue id](issue link)
* Change will also be needed in `<insert branch name here>`
* Future work required by [issue id](issue link)
* Documentation changes required in [issue id](issue link)
* Merge with/before/after `<insert PR URL here>`

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `<development system>`
  * Local development environment
  * Virtual Shasta

### Test description:

_How were the changes tested and success verified? If schema changes were part of this change, how were those handled in your upgrade/downgrade testing?_

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)?
- Were continuous integration tests run? If not, why?
- Was upgrade tested? If not, why?
- Was downgrade tested? If not, why?
- Were new tests (or test issues/Jiras) created for this change?

## Risks and Mitigations

_Are there known issues with these changes? Any other special considerations?_

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by rustydb — APPROVED** [review]
Seems simpler.

File: operations/node_management/Reboot_NCNs.md [hunk 1]
```diff
@@ -173,8 +173,8 @@ Before rebooting NCNs:
     5. If desired verify method of boot is expected. If the `/proc/cmdline` begins with `BOOT_IMAGE` then this NCN booted from disk:
 
        ```bash
-       ncn# egrep -o '^(BOOT_IMAGE.+/kernel)' /proc/cmdline
-       BOOT_IMAGE=(mduuid/a3899572a56f5fd88a0dec0e89fc12b4)/boot/grub2/../kernel
+       ncn# egrep -o '^(BOOT_IMAGE|kernel)' /proc/cmdline
+       BOOT_IMAGE
```
Comment (mharding-hpe) [inline operations/node_management/Reboot_NCNs.md:177]:
````suggestion
       ```

       Expected output is:

       ```text
       BOOT_IMAGE
````

**Review by mharding-hpe — CHANGES_REQUESTED** [review]
This needs a main branch PR too. Does it need 1.0 as well?

**Review by mharding-hpe — APPROVED** [review]
I think this is improved with my suggestion, but I'm fine with it merging either way. We do need to get a main branch PR and, if applicable, a 1.0 PR.

**Pull request merged**